examples/README.md

```diff
@@ -0,0 +1,29 @@
+# Rev.ai Node SDK Examples
+### Setup:
+From the `examples/` directory run `npm install`
+In the `config/` directory,  copy `sample.config.json` as `config.json`
```

Comment:
Opinion: this would be more clear as `In config/sample.config.json, insert your access token where specified and rename the file to config.json`